OpenKh.Bbs/SystemData/Command.cs

```diff
@@ -6,7 +6,7 @@ namespace OpenKh.Bbs.SystemData
 {
     public class Command
     {
-        public enum Type : ushort
+        public enum Kind : ushort
         {
             COMMAND_KIND_None = 0,
```

Comment:
Unlike C/C++, you could rename `COMMAND_KIND_None` to just `None` as you might need to do `Command.Kind.None` anyway to use it. Same for all the other enums you addded.

Reply:
Alright, that makes sense. I will change them whenever I get home.